jni/Makefile.am

```diff
@@ -0,0 +1,241 @@
+.PHONY: clean-precomp precomp
```

Comment:
Would it make sense to have a patch for original Makefile here instead of rewriting the whole file?

Reply:
This is an initial version with the copy of the whole files. Changes should be extracted to a script or diff file.